src/math/Euler.js

```diff
@@ -11,98 +11,90 @@ class Euler {
 
 		this.isEuler = true;
 
-		this._x = x;
-		this._y = y;
-		this._z = z;
-		this._order = order;
+		this.x = x;
+		this.y = y;
+		this.z = z;
+		this.order = order;
 
 	}
 
-	get x() {
+	get _x() {
```

Comment:
`_x`, `_y`, `_z` fields etc. are preserved as getters/setters to not break user code (library code no longer references these).

If these are considered internal, we can exclude them without deprecation.